src/Microsoft.AspNetCore.WebHooks.Receivers.Dropbox/Internal/DropboxServiceCollectionSetup.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.WebHooks.Filters;
+using Microsoft.AspNetCore.WebHooks.Metadata;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.AspNetCore.WebHooks.Internal
+{
+    /// <summary>
+    /// Methods to add services for the Dropbox receiver.
+    /// </summary>
+    public static class DropboxServiceCollectionSetup
+    {
+        /// <summary>
+        /// Add services for the Dropbox receiver.
+        /// </summary>
+        /// <param name="services">The <see cref="IServiceCollection"/> to update.</param>
+        public static void AddDropboxServices(IServiceCollection services)
+        {
+            if (services == null)
+            {
+                throw new ArgumentNullException(nameof(services));
+            }
+
+            services.TryAddEnumerable(ServiceDescriptor.Transient<IConfigureOptions<MvcOptions>, MvcOptionsSetup>());
+            services.TryAddEnumerable(ServiceDescriptor.Singleton<IWebHookMetadata, DropboxMetadata>());
+
+            services.TryAddSingleton<DropboxVerifySignatureFilter>();
+        }
+
+        private class MvcOptionsSetup : IConfigureOptions<MvcOptions>
```

Comment:
Turns out the lambda to create a `WebHookMvcOptionsAddFilter<TFilter> : IConfigureOptions<MvcOptions>` instance with a specific `Order` value is pretty ugly. These private classes are pretty straightforward and easy to use.